sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -165,8 +168,9 @@ export interface QueueOptions {
   duplicateDetectionHistoryTimeWindow?: string;
 
   /**
-   * The URL of Service Bus queue to forward deadlettered messages to.
-   *
+   * Absolute URI or relative path of the queue or topic messaging entity the dead-lettered
+   * messages are to be forwarded to.
+   * For e.g., sb://<your-service-bus-namespace-endpoint>/<queue-or-topic-name>
```

Comment:
Since we have 2 options (uri & name), but we are providing 1 example, it is better if we are explicit about what the example is for.
```suggestion
   * The absolute URI should be of the form sb://<your-service-bus-namespace-endpoint>/<queue-or-topic-name>
```